File: Clase7/tarea_clase7_libro.py
class libro:
    def __init__(self, titulo, autor,año):
        self.titulo = titulo
        self.autor = autor
        self.año = año

    def mostrar_info(self):
        print(f"Este es el titulo:{self.titulo}, Autor: {self.autor} y año:{self.año}.")

File: Clase7/tarea_clase7_libro.java
package Clase7;

public class tarea_clase7_libro {
    String titulo;
    String autor;
    int año;

    // Constructor
    public tarea_clase7_libro(String titulo, String autor, int año) {
        this.titulo = titulo;
        this.autor = autor;
        this.año = año;
    }

    // Método
    public void mostrar_info() {
        
        System.out.println("Este es el titulo: " + titulo + " autor: " + autor + " y año de educicion:" + año);
    }

}
